include/deal.II/base/numbers.h

```diff
@@ -892,7 +892,7 @@ namespace numbers
   {
     // Use the specialized definition for two ADOL-C taped types
     return value_is_less_than(value_1,
-                              internal::NumberType<adouble>::value(value_2));
+                              ::internal::NumberType<adouble>::value(value_2));
```

Comment:
(*oops* My bad - the regression tester had been out of commission for a week. I missed a reboot.)

The global namespace qualifier `::internal` is a typo. Fixed in #14162